Shared/Samples/Create and edit geometries/CreateAndEditGeometriesView.swift

```diff
@@ -226,10 +235,10 @@ extension GeometryEditorMenu {
 @MainActor
 class GeometryEditorMenuModel: ObservableObject {
     /// The geometry editor.
-    let geometryEditor: GeometryEditor
+    let geometryEditor = GeometryEditor()
     
     /// The graphics overlay used to save geometries to.
-    let graphicsOverlay: GraphicsOverlay
+    let graphicsOverlay = GraphicsOverlay(renderingMode: .dynamic)
```

Comment:
A better name for this might be `geometryOverlay`. You could go with `geometriesOverlay`, but it is well understood that the adjectival noun (geometry) could be referring to more than one geometry. For example, we say "car mechanic", not "cars mechanic", and we understand that the mechanic may work (and most likely works) on more than one car.